lib/tasks/seed_from_csv.rake

```diff
@@ -43,7 +27,7 @@ namespace :seed_from_csv do
 
     merchants_csv.each do |row|
       row[:id] = row[:id].to_i
-      Merchant.create(row)
+      Merchant.create!(row)
```

Comment:
Adding the bang (!) on the end of the create allowed for easier debugging since an error was thrown during rake file execution.